src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -137,8 +137,8 @@ class AuthScreens extends React.Component {
             App.reconnectApp();
         }
 
-        App.redirectThirdPartyDesktopSignIn();
         App.setUpPoliciesAndNavigate(this.props.session, !this.props.isSmallScreenWidth);
+        App.redirectThirdPartyDesktopSignIn();
```

Comment:
Is there a reason for this change?

Reply:
No intentional change. I was worried that the desktop redirect might interfere with the Rooms Navigation but I don't _think_ there's a scenario where that happens?